plugins/woocommerce-docs/tsconfig.json

```diff
@@ -22,6 +22,6 @@
 
 		// Skip type checking of declaration files because some libraries define two copies of the same type in an inconsistent way
 		"skipLibCheck": true,
-		"module": "esnext"
+		"module": "CommonJS"
```

Comment:
Out of curiosity -- why this change?

Reply:
This change is so we can use `import` in the manifest generation script which is a node.js TypeScript file. Since it doesn't really affect the client side code (we're just using TS for typechecking in the webpack) I have changed this to suit the script. Later we could remove this because the plan ultimately will be to move the manifest generation into monorepo utils or something like that.